c7n/resources/asg.py

```diff
@@ -476,6 +476,29 @@ def process(self, asgs, event=None):
                 len(a['Instances']) < a['MinSize']]
 
 
+@filters.register('progagated-tag')
+class PropagatedTagFilter(Filter):
+
+    schema = type_schema(
+        'progagated-tag',
+        key={'type': 'string'},
+        value={'type': 'string'}
+    )
+
+    def process(self, asgs, event=None):
+        key = self.data.get('key', DEFAULT_TAG)
+        value = self.data.get('value', None)
+        results = []
+        for r in asgs:
+            tags = r.get('Tags', [])
+            for t in tags:
+                if not t['PropagateAtLaunch']:
+                    continue
+                if t['Key'] == key and t['Value'] == value:
```

Comment:
people don't actually care about the value match in this context, they have standard value filter semantics to do that.